Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelUploadTests.swift

```diff
@@ -0,0 +1,130 @@
+import NetworkService
+import XCTest
+
+final class SitterProfileViewModelUploadTests: XCTestCase {
+    private var viewModel: SitterProfileViewModel!
+
+    override func setUp() {
+        super.setUp()
+        URLProtocol.registerClass(MockURLProtocol.self)
+        viewModel = SitterProfileViewModel()
+    }
+
+    override func tearDown() {
+        URLProtocol.unregisterClass(MockURLProtocol.self)
+        viewModel = nil
+        super.tearDown()
+    }
+
+    func testUploadSuccessfullyExecuted() async {
+        MockURLProtocol.requestHandler = { request in
+            let response = try XCTUnwrap(
+                HTTPURLResponse(
+                    url: XCTUnwrap(request.url),
+                    statusCode: 200,
+                    httpVersion: nil,
+                    headerFields: nil
+                )
+            )
+            return (response, nil)
+        }
+
+        do {
+            try await viewModel.upload()
+            XCTAssertEqual(viewModel.isErrorOccurred, false)
+        } catch {}
+    }
```

Comment:
Can we use throw function instead of using do/catch block?
Please review the code below as well.

```suggestion
    func testUploadSuccessfullyExecuted() async throws {
        MockURLProtocol.requestHandler = { request in
            let response = try XCTUnwrap(
                HTTPURLResponse(
                    url: XCTUnwrap(request.url),
                    statusCode: 200,
                    httpVersion: nil,
                    headerFields: nil
                )
            )
            return (response, nil)
        }

        try await viewModel.upload()
        XCTAssertEqual(viewModel.isErrorOccurred, false)
    }
```

Reply:
Fixed with 06c4f69

Reply:
What about other tests? Why did you fix only the first one?

Reply:
Fixed